server/src/Tests.h

```diff
@@ -471,6 +471,11 @@ namespace tests {
             FileInfo getFileByName(char fileName) const {
                 return filesValues.value()[fileName - 'A'];
             }
+            
+            std::string getError() const {
+                assert(!errorDescriptors.empty());
```

Comment:
Just return default value instead of crash program

Reply:
changed